Lib/test/test_typing.py

```diff
@@ -4496,13 +4618,6 @@ class Z(Generic[P]):
         self.assertEqual(G5.__parameters__, G6.__parameters__)
         self.assertEqual(G5, G6)
 
-    def test_var_substitution(self):
-        T = TypeVar("T")
-        P = ParamSpec("P")
```

Comment:
Where did this test go? I don't see a ParamSpec test in the new `test_var_substitution` test.

Reply:
Ah nevermind, I found it: moved to https://github.com/python/cpython/pull/27507/files#diff-04d29c98076c2d6bb75921ea9becb26a862544d39b71db87b6e354c759b9305dR580